src/api/app/controllers/webui/request_controller.rb

```diff
@@ -149,6 +149,12 @@ def beta_show
     @diff_limit = params[:full_diff] ? 0 : nil
     @diff_to_superseded_id = params[:diff_to_superseded]
     @actions = @bs_request.webui_actions(filelimit: @diff_limit, tarlimit: @diff_limit, diff_to_superseded: @diff_to_superseded, diffs: false)
+    reviews = @bs_request.reviews.where(state: 'new')
```

Comment:
Those new lines are duplicated. See lines 143 to 147.

Reply:
This still needs to be done. The lines are still there and duplicated.

Reply:
Done, I pushed a commit to handle this.

Reply:
Thanks. I've squashed the commit